plugins/search-react/src/components/SearchTracker/SearchTracker.tsx

```diff
@@ -23,14 +23,17 @@ import { useSearch } from '../../context';
  */
 export const TrackSearch = ({ children }: { children: React.ReactChild }) => {
   const analytics = useAnalytics();
-  const { term } = useSearch();
+  const { term, result } = useSearch();
+  const numberOfResults = result.value?.results.length ?? 0;
```

Comment:
I called out the max of 25 results in the linked issue. I should have mentioned that in this PR as well for clarity. I tried to figure out if we could get the total results but it doesn't seem possible (do you agree?)

For our use case (and I think probably others) the most useful part is seeing which searches return zero results, so this is fine. Although potentially the naming could be clearer in this case.

Reply:
>  I wonder if it would make sense to expose an optional numberOfResults (or similar) property on the search result interface (from the backend) and populate the value from the response, there?

If we can populate the actual numberOfResults (not just page 1) from the backend, that would be ideal. I'll have a look at that.

Thanks!

Reply:
PG would be awesome to get to, if you're able!  Thanks for digging in.

Reply:
Added PG - The thing about this is the query is currently limited to pageSize + 1 (defaults at 26) . I could update this [query](https://github.com/backstage/backstage/blob/9bc125eca134a04031b9a00a596b070c9be8a628/plugins/search-backend-module-pg/src/database/DatabaseDocumentStore.ts#L204) removing the limit but I assume it's been put in for good reason (performance?) so would like some guidance here.

Reply:
I've fixed some PG related tests but there's definitely still one which is failing related to not finding a DB. I'll need some pointers on this please :)

Reply:
> Added PG - The thing about this is the query is currently limited to pageSize + 1 (defaults at 26) . I could update this [query](https://github.com/backstage/backstage/blob/9bc125eca134a04031b9a00a596b070c9be8a628/plugins/search-backend-module-pg/src/database/DatabaseDocumentStore.ts#L204) removing the limit but I assume it's been put in for good reason (performance?) so would like some guidance here.

Mmmmm, yeah.  That's not ideal, and removing the limit is also not ideal.  It looks like there are [some mechanisms available](https://stackoverflow.com/questions/28888375/run-a-query-with-a-limit-offset-and-also-get-the-total-number-of-rows) to us to get the full count without returning all of them...  But maybe just to get this out, we could continue returning `undefined` for postgres, and just create a follow-up issue to add support?

Reply:
Sounds good - I've updated this PR and will create a follow up issue once this is merged?

Reply:
hey @philberryman, we're still not able to see the number of search results when using PG, do you know if this issue has been tackled already?